src/App.jsx

```diff
@@ -66,7 +66,10 @@ export function App() {
 							)
 						}
 					/>
-					<Route path="/list" element={<List data={data} />} />
+					<Route
+						path="/list"
+						element={listToken ? <List data={data} /> : <Navigate to="/" />}
+					/>
```

Comment:
It's a very useful line of code, @emilysellers. Thank you for doing this extra to make the app more usable. 😁👍🏻